lab_1_keywords_tfidf/main.py

```diff
@@ -80,7 +104,7 @@ def calculate_tf(frequencies: dict[str, int]) -> Optional[dict[str, float]]:
     Returns:
     dict: A dictionary with tokens and corresponding term frequency score
 
-    In case of corrupt input arguments, None is returned
+    In case of corrupt input   arguments, None is returned
```

Comment:
It is also advised not to alter docstrings in any way

Reply:
Done.